src/shared/components/PopoverBase/PopoverBase.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled'
+import { Placement } from '@popperjs/core'
+import Tippy from '@tippyjs/react/headless'
+import React from 'react'
+
+export type PopoverBaseProps = {
+  content: React.ReactNode
+  isVisible?: boolean
+  placement?: Placement
+  offset?: [number, number]
+  hideOnClick?: boolean
+  className?: string
+}
+
+export const PopoverBase: React.FC<PopoverBaseProps> = ({
```

Comment:
I think I'd opt for just `Popover` for this component name

Reply:
There is another component called Popover built on top coming that's why

Reply:
see PR popover

Reply:
I think we agreed to call this one `Popover` and the one from the other PR `PopoverDialog`, no?

Reply:
I think that might be misleading as in Figma it already is called `popover`

Reply:
We talked about this during our call last week :P The Figma component will get another name, it's temporary only. The Github issue you've linked mentions the correct names for the implementation

Reply:
hmm, true, since I got sick my memory got deleted lol